### ISSUE TITLE
Question: Webpage rendered through Voila doesn't respond to widget events

### ISSUE THREAD

**Issue by goelakash** [QuantStack/voila#246]
(I'd ask this on StackOverflow, but I didn't see a `Voila` tag for questions there, so asking here).
Hi,

Here's the source-code to my Jupyter notebook:
https://github.com/goelakash/Hyperparameter-Tuning-With-Voila/blob/master/MNIST_widgets.ipynb

In a nutshell, I'm trying to build a toy neural-net using MNIST dataset to demonstrate how messing with hyper-parameters changes the accuracy of the model.

Basic idea is to initialize a set of slider-widgets for the hyper-parameters, use their values as variables in other objects, and add an observer to each slider-widgets which updates the objects corresponding to the their values.

If I could summarise, this is the general setup:
```
# For illustration only
slider = widgets.IntSlider()
foo  = slider.value

def update_foo():
    foo = slider.value

def change_slider(change):
    slider.value = change.new
    update_foo()

slider.observe(change_slider, names='value')
```

Everything works fine* when I run the Jupyter notebook. Adjusting the sliders changes the attributes of the neural-net and training-loop params. Clicking on the buttons trains/tests the network as expected.

However, I'm unable to see any kind of response (initializing model, training and testing) when I change the sliders on the webpage rendered through Voila.

I'm sure there is a gap in my understanding of widgets here. Can you guys help me figure out how I can make this work through Voila?

(* The outputs are a bit ugly, but verbose, so its fine)

**Comment by SylvainCorlay** [comment]
@goelakash you seem to be making use of `print` statements, which have nowhere to go at the moment.

One way to achieve this with the current version is to create an `Output` widget and capture all of these outputs into it.

**Comment by goelakash** [comment]
Thanks, your advice worked. I bound all the `print` statments with the output widget and it works exactly as expected.

**Comment by goelakash** [comment]
This is how the updated dashboard looks like now:
https://raw.githubusercontent.com/goelakash/Hyperparameter-Tuning-With-Voila/master/images/dashboard.png
